thrift/src/main/thrift/client.thrift

```diff
@@ -315,6 +324,7 @@ struct TSRawDataQueryReq {
   6: required i64 statementId
   7: optional bool enableRedirectQuery;
   8: optional bool jdbcQuery;
+  9: optional i64 timeout
```

Comment:
Since you've added this fields in req remember to use that in ClientRPCServiceImpl instead of default timeout from config.

Reply:
This parameter will be used in future new methods.